config/kafka-monitor.properties

```diff
@@ -39,8 +39,11 @@
     "zookeeper.connect": "localhost:2181",
     "bootstrap.servers": "localhost:9092",
     "produce.record.delay.ms": 100,
-    "produce.topic.autoTopicCreationEnabled": true,
-    "produce.topic.autoTopicReplicationFactor" : 1,
+    "produce.topic.topicCreationEnabled": true,
+    "topic-management.replicationFactor" : 1,
+    "topic-management.partitionsPerBroker" : 2,
```

Comment:
I'm not sure.  Ratios don't need to be natural numbers, but the number of partitions per broker does need to be a natural number.